client/lib/post-normalizer/test/index.js

```diff
@@ -51,6 +19,36 @@ function asyncTransform( post, callback ) {
 }
 
 describe( 'post-normalizer', function() {
+	let normalizer, safeImageUrlFake, allTransforms;
+
+	require( 'lib/react-test-env-setup' ).auto();
+	require( 'test/quick-mock' )( __dirname );
```

Comment:
Let's skip that for now, and get back when we are about to add more tests to single test runner and we really need it :)